src/components/parameter/FieldSelectorModal.js

```diff
@@ -0,0 +1,76 @@
+import Button from '@material-ui/core/Button';
+import Container from '@material-ui/core/Container';
+import Dialog from '@material-ui/core/Dialog';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import { withStyles } from '@material-ui/core/styles';
+import PropTypes from 'prop-types';
+import React from 'react';
```

Comment:
I agree that the react import should be place first!
The reason it got moved below is because I used an import sorting feature on vscode and I liked how the imports were all sorted afterwards. I don't know if you've already discussed about this but what do you think of enforcing the import order style with eslint @hagoptaminian @pyphilia ? https://dev.to/otamnitram/sorting-your-imports-correctly-in-react-213m